teaspoon-qunit/spec/integration_spec.rb

```diff
@@ -0,0 +1,73 @@
+require_relative "./spec_helper"
+
+feature "Running in the browser", browser: true do
+  before do
+    path = File.expand_path('../../test/javascripts', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.